fmn/consumer/consumer.py

```diff
@@ -18,6 +18,39 @@
 import logging
 log = logging.getLogger("fmn")
 
+import pika
+connection = pika.BlockingConnection()
+
+
+def notify_prefs_change(openid):
+    import json
+    import pika
+    connection = pika.BlockingConnection()
+    msg_id = '2016-%s' % uuid.uuid4()
```

Comment:
I guess the year should be a variable that comes from a datetime.date.today().year?